src/Pages/Signup/SignupMain.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import SignupInput from './SignupInput';
+import SignupSelect from './SignupSelect';
+import SignupButton from './SignupButton';
+import SignupCheckBox from './SignupCheckBox';
+
+class SignupMain extends Component {
+  render() {
+    return (
+      <article className="SignupMain_article">
+        <SignupInput
+          placeholder="성"
+          className="signupInput"
+          text="성 필드는 필수 필드입니다."
+        />
+        <SignupInput placeholder="이름" className="signupInput" />
+        <SignupInput placeholder="생일" className="signupInput" />
+        <SignupInput placeholder="KR (+82)" className="signupInput" />
+
+        <SignupSelect
+          value="성별 (선택 사항)"
+          firstOption="남성"
+          secondOption="여성"
+          thirdOption="응답 거부"
```

Comment:
options
```suggestion
          options={['남성', '여성', '응답 거부']}
```
좀 더 적합한 자료형을 사용할 수 있을 것 같습니다